applications/SwimmingDEMApplication/python_scripts/variables_management.py

```diff
@@ -79,12 +79,13 @@ def AddExtraProcessInfoVariablesToFluidModelPart(self, parameters, fluid_model_p
         VariablesManager.AddFrameOfReferenceRelatedVariables(parameters, fluid_model_part)
 
         fluid_model_part.ProcessInfo.SetValue(Kratos.FRACTIONAL_STEP, 1)
-        gravity = Vector(3)
+
         if parameters["custom_fluid"]["body_force_on_fluid_option"].GetBool():
-            gravity[0] = parameters["GravityX"].GetDouble()
-            gravity[1] = parameters["GravityY"].GetDouble()
-            gravity[2] = parameters["GravityZ"].GetDouble()
-        fluid_model_part.ProcessInfo.SetValue(Kratos.GRAVITY, gravity)
+            gravity = Vector([self.project_parameters["gravity_parameters"]["direction"][i].GetDouble() for i in range(3)])
```

Comment:
```suggestion
            gravity = self.project_parameters["gravity_parameters"]["direction"].GetVector()
```

Reply:
Thanks! I will apply all of these tomorrow